security/production-secure-deploy/internal-rolebindings/cfrb-internal-connect-0.yaml

```diff
@@ -0,0 +1,18 @@
+# rolebinding `internal-connect-0` gives user `connect` the role 
+# `SecurityAdmin` for cluster `connect` in namespace `confluent`
+apiVersion: platform.confluent.io/v1beta1
+kind: ConfluentRolebinding
+metadata:
+  name: internal-connect-0
```

Comment:
we should not use the `internal` prefix, this will coincide with the role binding created by CFK.

Reply:
Yeah, it might be better to use general name instead of the same name we used for internal rolebindings. And some CR specify topic name with some internal topic name, it might be better to change also.